src/test/groovy/com/coxautodev/graphql/tools/EndToEndSpec.groovy

```diff
@@ -93,18 +93,16 @@ class EndToEndSpec extends Specification {
 
                 @Override
                 void onError(Throwable t) {
-
                 }
 
                 @Override
                 void onComplete() {
-
                 }
             })
             latch.await(3, TimeUnit.SECONDS)
 
         then:
-            returnedItem.id == 1
+            returnedItem.get("onItemCreated").id == 1
```

Comment:
Not 100% sure if this is correct.

Reply:
What prompted the change?

Reply:
`returnedItem.id` was null and of course the junit failed. Does it look correct?

Reply:
@apottere - are you happy enough with this?

Reply:
I dug into this - it looks like the reason the behavior change was due to this change in graphql-java: 

https://github.com/graphql-java/graphql-java/commit/adcd3f7e8248f7a591e2f8110ff0aa1910b341f9#diff-83f68d2746cff37c74d371d07db0e53b

"This commit adds the root field to subscription results to conform to
the current version of the GraphQL specification."

So it's an expected change to be spec compliant.